src/biogeophys/CanopyStateType.F90

```diff
@@ -52,6 +52,9 @@ module CanopyStateType
 
      real(r8) , pointer :: dleaf_patch              (:)   ! patch characteristic leaf width (diameter) [m]
                                                           ! for non-ED/FATES this is the same as pftcon%dleaf()
+     integer ,  pointer :: wesley_pft_index_patch   (:)   ! Wesley PFT index for FATES drydep calculations
+     integer ,  pointer :: drydep_season_patch      (:)   ! Season for dry deposition calculations
```

Comment:
Can you name it `wesley_veg_index_patch` and `wesley_season_index_patch` for consistency? And also in comments have `dry deposition` spelled out everywhere?

Reply:
Sure. Agree that would add better name spacing symmetry...